pyfstat/grid_based_searches.py

```diff
@@ -131,7 +131,7 @@ def __init__(
     def _set_output_keys(self):
         self.output_keys = self.search_keys.copy()
         self.output_keys.append("twoF")
-        if hasattr(self.search, "twoFX"):
+        if self.search.singleFstats:
```

Comment:
Let me check if I got this right: `singleFstats` is the boolean label.

Reply:
Yes. I named it for parallelism with CFSv2's `outputSingleFstats`, but dropping the verb since we're not necessarily outputting anything here...  
By contrast, the results are `self.twoFX`.

This whole mess could be reconsidered by making the attributes more hierarchical, e.g. having two dicts `self.results` and `self.options`. But I guess that falls under "PyFstat 2.0".

Reply:
Yes